notebooks/tests/test_tellseq_D_variable_volume_pooling.py

```diff
@@ -0,0 +1,60 @@
+import unittest
```

Comment:
Looking at this file, I think it is not flake8 compatible, which was surprising to me but I also might be wrong; and I think the reason is not being checked is because this line does _not_ run flake8 on the notebooks folder: https://github.com/biocore/kl-metapool/blob/master/.github/workflows/ci.yml#L137 ... can you make sure that you add it? I think adding `notebooks/tests/` should suffice.